Ghidra/Framework/Utility/src/main/java/ghidra/util/config/ColorHexConvert.java

```diff
@@ -0,0 +1,80 @@
+/* ###
+ * IP: GHIDRA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ghidra.util.config;
+
+import java.awt.*;
+
+public class ColorHexConvert {
```

Comment:
Do you have any actual reason of not using `Color.decode` but write one yourself?

Reply:
`Color.decode` does not support transparent translation，just support RGB

Reply:
Ok. Then I propose a better name. What about use `ColorFormat` as the class name?